.github/workflows/ci.yml

```diff
@@ -359,6 +359,7 @@ jobs:
       uses: ./source/.github/actions/android-ci
       with:
         arch: ${{matrix.arch}}
+        vtk_version: v9.3.0
```

Comment:
why not commit ?

Reply:
Vtk::token

Reply:
ha right

Reply:
please add a link to the issue on top of the step so that we can fix it when we can